README.md

```diff
@@ -36,6 +37,7 @@ $ apt-get install libgraphviz-dev
 $ pip install pygraphviz \
      --install-option="--include-path=/usr/include/graphviz" \
      --install-option="--library-path=/usr/lib/graphviz/"
+$ pip install websockets
```

Comment:
Does it make sense to add a section in the readme about how to start up / use the GUI?

Reply:
GUI itself is not usable yet. Let me do that when GUI is integrated in test_driver.py